spec/index.bs

```diff
@@ -610,36 +614,32 @@ dictionary GPUObjectDescriptorBase {
 
 ## Invalid Internal Objects &amp; Contagious Invalidity ## {#invalidity}
 
-If an object is successfully created, it is <dfn dfn>valid</dfn> at that moment.
-An [=internal object=] may be <dfn dfn>invalid</dfn>.
-It may become [=invalid=] during its lifetime, but it will never become valid again.
+Object creation operations in WebGPU are internally asynchronous, so they don't fail with exceptions.
+Instead, returned objects may refer to [=internal objects=] which are either
+<dfn dfn>valid</dfn> or <dfn dfn>invalid</dfn>.
+An object may become [=invalid=] during its lifetime, but it will never become [=valid=] again.
```

Comment:
Maybe be more concrete about this. Instead of saying that any object can be invalid, we should say that some object types allow objects to be come invalid, while most are valid permanently since creation.